support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/utils/ophan.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import { trackComponentEvents } from 'helpers/tracking/ophan';
+import { type PaymentMethod } from 'helpers/paymentMethods';
+import { type ContributionType } from 'helpers/contributions';
+
+export const OPHAN_COMPONENT_ID_SIGN_IN = 'sign-into-the-guardian-link';
+export const OPHAN_COMPONENT_ID_SIGN_UP = 'set-password';
+export const OPHAN_COMPONENT_ID_SET_REMINDER = 'reminder-test-link-clicked';
+export const OPHAN_COMPONENT_ID_MARKETING = 'marketing-permissions';
+export const OPHAN_COMPONENT_ID_SURVEY = 'contribution-thankyou-survey';
+export const OPHAN_COMPONENT_ID_SOCIAL_FACEBOOK =
+  'contribution-thankyou-social-facebook';
+export const OPHAN_COMPONENT_ID_SOCIAL_TWITTER =
+  'contribution-thankyou-social-twitter';
+export const OPHAN_COMPONENT_ID_SOCIAL_LINKED_IN =
+  'contribution-thankyou-social-linked-in';
+export const OPHAN_COMPONENT_ID_SOCIAL_EMAIL =
+  'contribution-thankyou-social-email';
+export const OPHAN_COMPONENT_ID_AUS_MAP = 'contribution-thankyou-aus-map';
+
+export const OPHAN_COMPONENT_ID_READ_MORE_SIGN_IN =
+  'contribution-thankyou-read-more-sign-in';
+export const OPHAN_COMPONENT_ID_READ_MORE_SIGN_UP =
+  'contribution-thankyou-read-more-sign-up';
+export const OPHAN_COMPONENT_ID_READ_MORE_SET_REMINDER =
+  'contribution-thankyou-read-more-set-reminder';
+
+const OPHAN_EVENT_ID_PAYMENT_METHOD = 'contributions-thankyou-payment-method';
+const OPHAN_EVENT_ID_CONTRIBUTION_TYPE =
+  'contributions-thankyou-contribution-type';
+const OPHAN_EVENT_ID_SIGNED_IN = 'contributions-thankyou-signed-in';
+const OPHAN_EVENT_ID_KNOWN_EMAIL = 'contributions-thankyou-known-email';
+
+const trackPaymentMethod = (paymentMethod: PaymentMethod) => {
+  trackComponentEvents({
+    component: {
+      componentType: 'ACQUISITIONS_OTHER',
+    },
+    action: 'VIEW',
+    id: OPHAN_EVENT_ID_PAYMENT_METHOD,
+    value: paymentMethod,
+  });
+};
+
+const trackContributionType = (contributionType: ContributionType) => {
+  trackComponentEvents({
+    component: {
+      componentType: 'ACQUISITIONS_OTHER',
+    },
+    action: 'VIEW',
+    id: OPHAN_EVENT_ID_CONTRIBUTION_TYPE,
+    value: contributionType,
+  });
+};
+
+const trackSignedIn = (isSignedIn: boolean) => {
+  trackComponentEvents({
+    component: {
+      componentType: 'ACQUISITIONS_OTHER',
+    },
+    action: 'VIEW',
+    id: OPHAN_EVENT_ID_SIGNED_IN,
+    value: isSignedIn.toString(),
+  });
+};
+
+const trackKnownEmail = (isKnownEmail: boolean) => {
+  trackComponentEvents({
+    component: {
+      componentType: 'ACQUISITIONS_OTHER',
+    },
+    action: 'VIEW',
+    id: OPHAN_EVENT_ID_KNOWN_EMAIL,
+    value: isKnownEmail.toString(),
+  });
+};
+
+export const trackUserData = (
```

Comment:
the old thank you page should also send this data

Reply:
yeah good spot - just added this!